tvb_epilepsy/base/model/model_configuration.py

```diff
@@ -43,8 +43,14 @@ def __init__(self, yc=YC_DEF, Iext1=I_EXT1_DEF, Iext2=I_EXT2_DEF, K=K_DEF, a=A_D
 
         self.connectivity_matrix = connectivity_matrix
 
+        if n_regions is None:
```

Comment:
Param n_regions seems to be used as a boolean, in a not so nice manner (is /not it None).
The name n_regions, suggests expecting a number.
Either rename it, or use it as number (perhaps with default 0). I would recommend the second....